components/molecules/m-product-carousel.vue

```diff
@@ -9,15 +9,13 @@
         :max-rating="product.rating.max"
         :score-rating="product.rating.score"
         :link="product.link"
+        :wishlist-icon="false"
         link-tag="a"
-        :is-on-wishlist="isOnWishlist(product.data)"
-        @click:wishlist="toggleWishlist(product.data)"
       />
```

Comment:
`product.data` key can be also removed from `carouselProducts` computed property.

Reply:
done